packages/vite-plugin-cloudflare/src/index.ts

```diff
@@ -1080,6 +1080,39 @@ if (import.meta.hot) {
 				}
 			},
 		},
+		// Plugin to handle cron/email/etc triggers
+		{
+			name: "vite-plugin-cloudflare:trigger-handlers",
```

Comment:
```suggestion
			name: "vite-plugin-cloudflare:trigger-handlers",
			enforce: "pre",
```

We need `enforce: "pre"` here so that this runs ahead of the other middleware.

Reply:
✅